lib/src/configuration.dart

```diff
@@ -39,14 +39,23 @@ class Configuration {
   /// [readOnly] controls whether a [Realm] is opened as readonly.
   /// This allows opening it from locked locations such as resources,
   /// bundled with an application.  The realm file must already exists.
-  Configuration(List<SchemaObject> schemaObjects, {bool readOnly = false})
+  ///
+  /// [inMemory] specifies if this Realm should be opened in-memory.
```

Comment:
Only this one to stay
/// [inMemory] specifies if this Realm should be opened in-memory.

Reply:
done